File: src/main/java/com/lanka_guide/tripplanner/model/activity/transit/FlightTransit.java
package com.lanka_guide.tripplanner.model.activity.transit;

import com.lanka_guide.tripplanner.model.place.transport.Airport;

import java.time.ZonedDateTime;

public class FlightTransit extends Transit {
    Airport airport;

    public FlightTransit(Airport airport, ZonedDateTime from, ZonedDateTime to) {
        super(from, to);
        this.airport = airport;
    }
}

File: src/main/kotlin/com/lankaguide/tripplanner/model/activity/transit/FlightTransit.kt
package com.lankaguide.tripplanner.model.activity.transit

import com.lankaguide.tripplanner.model.place.transport.Airport

import java.time.ZonedDateTime

class FlightTransit(internal var airport: Airport, from: ZonedDateTime, to: ZonedDateTime) : Transit(from, to)
